### PR TITLE
Remove unused templates and use image template for more images

### LINKED ISSUE #424: Clean up codebase

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Remove unused templates and JS
- Use image template for more images
- Remove (required) from the contact-us form

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to all the pages and see that everything looks and works as expected

## Issue / Card

Fixes #424 

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

**Comment by solazio** [comment]
https://charmhub-io-425.demos.haus/

File: templates/404.html [hunk 1]
```diff
@@ -7,8 +7,17 @@
 {% block content %}
 <div class="p-strip is-deep">
   <div class="row">
-    <div class="col-4">
-      <img src="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg" alt="Error owl" width="400" height="400" class="p-image--smaller-mobile"/>
+    <div class="col-4 u-align--center">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg",
+          alt="Error owl",
```
Comment (jkfran) [inline templates/404.html:14]: In this case, is not an owl.
```suggestion
          alt="Error 404",
```

File: templates/500.html [hunk 2]
```diff
@@ -8,7 +8,16 @@
 <div class="p-strip is-deep">
   <div class="row">
     <div class="col-4">
-      <img src="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg" alt="Error owl" width="400" height="400" class="p-image--smaller-mobile"/>
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg",
+          alt="Error owl",
```
Comment (jkfran) [inline templates/500.html:14]: Same, but maybe we should change also the image? The current image is for a 404

https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg

```suggestion
          alt="Server error",
```

**Review by jkfran — APPROVED** [review]

File: templates/500.html (hunk 2 above)
Comment (solazio) [reply templates/500.html:14]: Will talk to @danielmutis today about a new image, but for now I'll stick to this one as we use the same on canonical.com

**Pull request merged**